communication/slack-config/channels.yaml

```diff
@@ -116,7 +116,6 @@ channels:
   - name: emea-users
   - name: eraser
   - name: es-users
-  - name: etcd
```

Comment:
Is this a rename operation?  Just want to ensure the channel doesn't get deleted as there is a lot of historical conversation in the channel.

Reply:
yeah, you need to scroll down.  For manageability, channels are alphabetical